### PR TITLE
Update papercuts events

### PR DESCRIPTION
**Pull request by assuntad23**
Made some changes to papercut event, based on rebranding efforts.

### CODE REVIEW

File: content/events/cofests/papercuts/main.md [hunk 1]
```diff
@@ -3,55 +3,47 @@ title: "Papercuts CoFests!"
 noun: "Papercuts CoFest"
 ---
 
-Papercuts events are [Galaxy CoFests](/events/cofests/) that address annoying, but easy to fix bugs ("paper cuts").  *Papercut CoFests* run for 24 hours (following the sun) and are collaborative events, dedicated to
+Papercuts events are [Galaxy CoFests](/events/cofests/) that address annoying, but easy to fix bugs ("paper cuts"). _Papercut CoFests_ run for 24 hours (following the sun) and are collaborative events, dedicated to
```
Comment (beatrizserrano) [inline content/events/cofests/papercuts/main.md:6]:
```suggestion
Papercuts events are [Galaxy CoFests](/events/cofests/) that address annoying, but easy to fix bugs ("paper cuts"). _Papercuts CoFests_ run for 24 hours (following the sun) and are collaborative events, dedicated to
```

File: content/events/cofests/papercuts/main.md [hunk 2]
```diff
@@ -3,55 +3,47 @@ title: "Papercuts CoFests!"
...
+-   **Before**
+    -   Each Papercuts event page has links to issues. in different parts of Galaxy, that have been flagged by experienced community members as being "fixable during a Papercuts."
+    -   Figure out which of these projects interest you the most.
+    -   Do some (even a little bit is useful) on that project.
+    -   Check the event page for more information on that particular event. Depending on what issues we're hoping to tackle, there may be some scheduled Zoom calls or collaboration materials.
+-   **During**
+    -   Join the discussion on [Gitter](https://gitter.im/galaxyproject/Lobby).
+    -   People on Gitter will provide guidance and resources, and help connect you to folks who specialize in the area you are interested in.
+    -   Ask for help! Collaborate! Work together!
+    -   Join the GTN CoFest Calls! (Or any other calls that )
```
Comment (beatrizserrano) [inline content/events/cofests/papercuts/main.md:34]: there is something missing here

File: content/events/cofests/papercuts/main.md [hunk 3]
```diff
@@ -3,55 +3,47 @@ title: "Papercuts CoFests!"
...
+-   **After**
+    -   Stay involved. Galaxy has lots of ways to get involved. Leverage your newly acquired knowledge to ask for help, and to help others.
+    -   Get your fixes integrated! You may or may not get in a fix integrated during the official Papercuts day. If you don't, then stick with it until you have the final polish on it, so it can be merged.
+    -   Join us again for the next Papercuts.
+
+# Guide for WGs: Hosting a Papercut event
```
[the remark this answers is not in the document]
Comment (assuntad23) [reply content/events/cofests/papercuts/main.md]: thank you! :)

**Comment by assuntad23** [comment]
Look good now, @beatrizserrano ?

**Review by beatrizserrano — APPROVED** [review]

**Comment by beatrizserrano** [comment]
> Look good now, @beatrizserrano ?

yes, sorry I missed it!

**Pull request merged**